### ISSUE TITLE
generator loss function calculation

### ISSUE THREAD

**Issue by SZH1230456** [zhangzx-sjtu/LANTERN-NeurIPS-2019#1]
I'm confused about the generator loss. In the paper, the cross entropy of policy is supposed to add into the target in the next code. However, it only includes the target while not adding the regularization part. Please explain it for a while?
![image](https://user-images.githubusercontent.com/38172534/84590349-3d824200-ae68-11ea-9119-fd2d6fbeb303.png)

**Comment by zhangzx-sjtu** [comment]
Error fixed and thanks for pointing it out : )

**Comment by zhangzx-sjtu** [comment]
Fixed the error and thanks for pointing it out. The regularization term is included with a weight 0.001 in our experiments for producing the results. Sorry for the confusion.
